bin/csv-filter

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'csv'
+
+# Returns an array of converter names.
+# \String +s+ contains a comma-separated list of converter names;
+# each optionally begins with a colon.
+def parse_converters(s)
+  converters = []
+  s.split(',').each do |name|
+    name.sub(/^:/, '')
+    sym = name.to_sym
+    converters.push(sym)
+  end
+  converters
+end
+
+options = {}
+
+OptionParser.new do |p|
+
+  p.program_name = File.basename $0
+  p.version = CSV::VERSION
+  p.release = nil
+  p.summary_indent = ' ' * 4
+  p.banner = <<-EOF
```

Comment:
Could you use `BANNER` not `EOF` for here document mark?
`EOF` (End Of File?) is difficult to understand.

Reply:
Done.